.gitignore

```
@@ -99,3 +99,12 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+#VSCODE IDE
+.vscode/*
+
+#module specific artifacts
+enrollments.json
+
+#OS artifacts
+.DS_Store
```

missing new line at end of file:

```suggestion
.DS_Store
```